### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by jermanuts** [beerisgood/Smartphone_Security#2] [General]
https://www.scss.tcd.ie/Doug.Leith/pubs/gboard_kamil.pdf

**Comment by beerisgood** [comment]
Thanks! Added
https://github.com/beerisgood/Smartphone_Security/commit/6f4afef485ae4c209d6ab63bf0861202f0207cb4

**Comment by jermanuts** [comment]
https://old.reddit.com/r/PrivacyGuides/comments/x32zj0/waydroid_or_grapheneos/imn1tou/

**Comment by beerisgood** [comment]
Added: https://github.com/beerisgood/Smartphone_Security/commit/8036379ab152b4946eb7ecca47d74970201a757d

**Comment by jermanuts** [comment]
https://twitter.com/mysk_co/status/1579997801047822336 there's more than that on there profile, such a mess.

https://mullvad.net/en/blog/2022/10/10/android-leaks-connectivity-check-traffic/
https://privsec.dev/posts/android/android-vpn-leakage-with-secondary-user-profiles/

**Comment by beerisgood** [comment]
While I already have VPN stuff included in https://github.com/beerisgood/Security-link-collection I added your links.
Thanks!

**Comment by quackerex** [comment]
White papers' from Apple

- Sign in with Apple: https://www.apple.com/privacy/docs/Sign_in_with_Apple_White_Paper_Nov_2019.pdf
- Private Relay: https://www.apple.com/icloud/docs/iCloud_Private_Relay_Overview_Dec2021.pdf
- Building a Trusted Ecosystem for Millions of Apps: https://www.apple.com/privacy/docs/Building_a_Trusted_Ecosystem_for_Millions_of_Apps.pdf
- Safari Privacy Overview: https://www.apple.com/safari/docs/Safari_White_Paper_Nov_2019.pdf
- Apple T2 Security Chip: https://www.apple.com/mideast/mac/docs/Apple_T2_Security_Chip_Overview.pdf
- Location Services Privacy Overview: https://www.apple.com/privacy/docs/Location_Services_White_Paper_Nov_2019.pdf
- ~~M1 Overview: https://www.apple.com/euro/business/mac/pdf/Apple-at-Work-M1-Overview.pdf~~

Apple Security Research
- https://security.apple.com

Lockdown Mode

- https://support.apple.com/en-us/HT212650

**Reply by beerisgood** [reply]
Added most of them. Thanks!

**Comment by jermanuts** [comment]
https://grapheneos.org/articles/positon-location-service

**Reply by beerisgood** [reply]
Added in https://github.com/beerisgood/Mobile_Security/commit/7871b4c0da77696c56b51fb65146f710e666f966 🍺
